src/main/java/seedu/address/model/person/Person.java

```diff
@@ -20,17 +20,19 @@ public class Person {
     // Data fields
     private final JobCode jobCode;
     private final Tag tag;
+    private final Remark remark;
 
     /**
      * Every field must be present and not null.
      */
-    public Person(Name name, Phone phone, Email email, JobCode jobCode, Tag tag) {
-        requireAllNonNull(name, phone, email, jobCode, tag);
+    public Person(Name name, Phone phone, Email email, JobCode jobCode, Tag tag, Remark remark) {
+        requireAllNonNull(name, phone, email, jobCode, tag, remark);
```

Comment:
You might want to consider making the remark field optional for the next iteration :D